pkg/bazel/bazel.go

```diff
@@ -28,7 +28,6 @@ import (
 	"aspect.build/cli/bazel/flags"
 	"aspect.build/cli/pkg/bazel/workspace"
 	"aspect.build/cli/pkg/ioutils"
-
```

Comment:
Unnecessary change. We actually like that first-party imports are a separate block from third-party block.

Reply:
Reverted.